usage/usageplot.py

```diff
@@ -1,26 +1,22 @@
-"""Create a plot of the lsst-dev node useage over time and prints the total 
+"""Create a plot of the lsst-dev node useage over time and prints the total
 node_hours of the jobs provided to usage.py.
 
 Code uses output from usage.py and needs to be assigned a plot title and a
 file name for the .png file to be made.
 """
 
-
 import matplotlib
+matplotlib.use('Agg')
 import matplotlib.pyplot as plt
 import numpy as np
 import argparse
 
 
-matplotlib.use('Agg')
-
-
 def create_parser():
     """Command line parser to pick up the command line arguments.
-
```

Comment:
Put this blank line back. Short summary must be separated by it from the rest of a docstring's body.